android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Thu Oct 29 18:09:13 CET 2020
```

Comment:
I'm not super convinced these gradle/gradle-wrapper files are necessary for a library. Did you mean to commit them?